Action/Tool/Message/OidcInitiationAction.php

```diff
@@ -62,8 +62,9 @@ public function __invoke(Request $request): Response
 
         } catch (LtiExceptionInterface $exception) {
             $this->logger->error(sprintf('OidcInitiationAction: %s', $exception->getMessage()));
+            $contentFromException = htmlspecialchars($exception->getMessage(), ENT_QUOTES);
 
-            return new Response($exception->getMessage(), Response::HTTP_BAD_REQUEST);
+            return new Response($contentFromException, Response::HTTP_BAD_REQUEST);
```

Comment:
I'm afraid https://github.com/oat-sa/bundle-lti1p3/blob/master/Action/Platform/Message/OidcAuthenticationAction.php#L65 suffers from the exactly same problem.